packages/mcdoc/src/runtime/checker/index.ts

```diff
@@ -965,18 +965,13 @@ export function simplify<T>(
 						node: initial,
 					}]
 					for (const entry of index.accessor) {
-						// TODO initial check against %parent and %key is to work around an mcdoc binder bug, it should always be an object with keyword property
-						if (
-							entry === '%parent' ||
-							(typeof entry != 'string' && entry.keyword === 'parent')
-						) {
+						if (typeof entry != 'string' && entry.keyword === 'parent') {
 							possibilities = possibilities.map(n => {
 								const node = n.parents.pop()
 								return { parents: n.parents, node }
 							})
 						} else if (
-							entry === '%key' ||
-							(typeof entry != 'string' && entry.keyword === 'key')
+							typeof entry != 'string' && entry.keyword === 'key'
 						) {
```

Comment:
Can this be one line now or does the formatter not like that?

Reply:
Oh also it probably should be `typeof entry !== 'string'` (instead of `!=` with one `=`)

Reply:
Formatter doesn't like it. I know it also bugged me a bit. Issue would go away when we split this in `simplifyIndexed`.

Reply:
Not sure what `resolveIndex` would do, and what the diference would be, regardless splitting it off would be a good idea.

To avoid bigger conflicts with my wip experiment to simplify the graph structure, it would be good to avoid bigger structural changes for now though.